app/helpers/lookup_keys_helper.rb

```diff
@@ -79,12 +79,12 @@ def can_edit_params?
   def lookup_key_with_diagnostic(obj, lookup_key, lookup_value)
     value, matcher = value_matcher(obj, lookup_key)
     inherited_value = lookup_key.value_before_type_cast(value)
-    overridden  = inherited_value.present? || lookup_value.value.present?
-    warnings  = lookup_key_warnings(lookup_key.required, overridden)
+    effective_value = lookup_value.id.nil? ?  inherited_value.to_s : lookup_value.value.to_s
```

Comment:
`to_s` is called on the values so that `false` will be turned into `"false"` and make `.present?` true.